src/main/java/pl/kinga/leczenie/specyfik/SpecyfikController.java

```diff
@@ -0,0 +1,47 @@
+package pl.kinga.leczenie.specyfik;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import pl.kinga.leczenie.sposobleczenia.SposobLeczenia;
+import pl.kinga.leczenie.sposobleczenia.SposobLeczeniaRepository;
+
+import java.util.Optional;
+
+@Controller
+public class SpecyfikController {
+
+    SposobLeczeniaRepository sposobLeczeniaRepository;
+    SpecyfikRepository specyfikRepository;
+
+    public SpecyfikController(SposobLeczeniaRepository sposobLeczeniaRepository, SpecyfikRepository specyfikRepository) {
+        this.sposobLeczeniaRepository = sposobLeczeniaRepository;
+        this.specyfikRepository = specyfikRepository;
+    }
+
+    @GetMapping("/SposobLeczeniaZnajdz")
```

Comment:
adresy endpointów troszkę inaczej się tworzy, ale to poznamy w lekcji którą dzisiaj udostępniłem